backend/test/api/base_test.clj

```diff
@@ -1,13 +1,18 @@
-(ns api.join-game-test
+(ns api.base-test
   (:require [expectations.clojure.test :refer :all]
             [clojure.test :as ctest]
             [mocking :as mocking]
             [api.base :as base]
-            [configs.hands :as hands]
```

Comment:
Wait, this is used below, right? as `hands/default-hand`?

Reply:
Probably that's why it fails the tests (sorry, couldn't run them).
Still... I don't understand how I removed this line... Thanks.